packages/store/src/internals.ts

```diff
@@ -1,11 +1,32 @@
-import { META_KEY } from './symbols';
+import { META_KEY, ActionOptions } from './symbols';
+import { Observable } from 'rxjs';
+
+export interface ObjectKeyMap<T> {
+  [key: string]: T;
+}
+
+export interface StateKlass {
+  [META_KEY]?: MetaDataModel;
+}
+
+type StateKeyGraph = ObjectKeyMap<string[]>;
+
+export interface ActionHandlerMetaData {
+  fn: string;
+  options: ActionOptions;
+  type: string;
+}
+
+export type GetStateFn<T> = () => T;
+export type SetStateFn<T> = (newState: T) => void;
+export type DispatchFn = (actions: any | any[]) => Observable<any>;
 
 export interface MetaDataModel {
   name: string;
-  actions: any;
+  actions: ObjectKeyMap<ActionHandlerMetaData[]>;
   defaults: any;
   path: string;
-  children: any[];
+  children: StateKlass[];
```

Comment:
Since its just just `Class`, you can problably make this `StateClass`

Reply:
Yipee! I was trying to keep with the naming of the Klass I found in the variable names. Would much prefer the name `StateClass`.